src/internal/cache-client.ts

```diff
@@ -1,4 +1,5 @@
 import {cache} from '@gomomento/generated-types';
+import cache_types = cache.cache_client;
```

Comment:
Why is it called `cache_types`? I think something like `grpc_cache_client` is clearer?

Reply:
Because it's from "generated-types", but yeah, they aren't all types.

I want to avoid "client". `grpc_cache` and `grpc_control`?

Reply:
yup that works.